L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCMulticlusters.cc

```diff
@@ -9,7 +9,7 @@ class HGCalTriggerNtupleHGCMulticlusters : public HGCalTriggerNtupleBase {
   HGCalTriggerNtupleHGCMulticlusters(const edm::ParameterSet& conf);
   ~HGCalTriggerNtupleHGCMulticlusters() override{};
   void initialize(TTree&, const edm::ParameterSet&, edm::ConsumesCollector&&) final;
-  void fill(const edm::Event& e, const HGCalTriggerNtupleEventSetup& es) final;
+  void fill(const edm::Event& e, const edm::EventSetup& es) final;
```

Comment:
There is an issue in conflict resolving when rebasing to the latest devel branch; this is reverting back to older version of the code.
Should not change this line. The usage of `EventSetup` has changed.